data/locale/en-US.ini

```diff
@@ -14,6 +14,7 @@ CustomCSS="Custom CSS"
 CSSFileReset="Reset CSS file path"
 CustomJS="Custom JavaScript"
 JSFileReset="Reset JS file path"
+EnvironmentVariables="Environment Variables"
```

Comment:
The German word for “Environment Variables“ is “Umgebungsvariablen”. You might want to add that as well.